README.md

```diff
@@ -39,10 +39,22 @@ $ ./gradlew docker
 
 You can run the fat jar as follows.
 
+
+```console
+$ java -jar scalar-admin-<version>-all.jar -c <COMMAND> -a <IP1:Port1,IP2:Port2,...>
+```
+
+**NOTE** If you previously used the following command, we recommend using the `-a` option instead since SRV has a known issue of returning asynchronous results. We will deprecate the `-s` option in the next major version.
```

Comment:
Should we return a warning message when that option is specified?

Reply:
Sounds good! I agree with returning a warning message.

Reply:
@feeblefakie 
@kota2and3kan 

Thanks for the advice.

I created another PR for this at https://github.com/scalar-labs/scalar-admin/pull/51

We can discuss it there 🙇